ensembl_rest_testing.conf

```diff
@@ -12,6 +12,10 @@ jsonp=1
 
 <Model::Registry>
   skip_initation = 1
+ preload = 1 
```

Comment:
do we still need the preload?
and might be cleaner if we remove the lines that have been commented out